src/main/kotlin/org/opensearch/commons/alerting/action/SearchCommentRequest.kt

```diff
@@ -0,0 +1,33 @@
+package org.opensearch.commons.alerting.action
+
+import org.opensearch.action.ActionRequest
+import org.opensearch.action.ActionRequestValidationException
+import org.opensearch.action.search.SearchRequest
+import org.opensearch.core.common.io.stream.StreamInput
+import org.opensearch.core.common.io.stream.StreamOutput
+import java.io.IOException
+
+class SearchCommentRequest : ActionRequest {
+
+    val searchRequest: SearchRequest
+
+    constructor(
+        searchRequest: SearchRequest
+    ) : super() {
+        this.searchRequest = searchRequest
```

Comment:
Should the `.indices(SYSTEM_INDEX_NAMES)` be added here, or do they get added elsewhere?

Reply:
That append is done in the RestHandler for SearchAlertingComments